presto-hive/src/main/java/io/prestosql/plugin/hive/HiveStorageFormat.java

```diff
@@ -99,6 +99,7 @@
     private final String serde;
     private final String inputFormat;
     private final String outputFormat;
+    @SuppressWarnings("ImmutableEnumChecker")
     private final DataSize estimatedWriterSystemMemoryUsage;
```

Comment:
We can have another PR to airlift to add `@Immutable` over `DataSize` class.

Reply:
Make this a TODO